scriptmodules/emulators/basilisk.sh

```diff
@@ -46,5 +46,5 @@ function configure_basilisk() {
     
     mkUserDir "$configdir/macintosh"
 
-    addSystem 1 "$md_id" "macintosh" "$md_inst/bin/BasiliskII --rom $romdir/macintosh/mac.rom --disk $romdir/macintosh/disk.img --config $configdir/macintosh/basiliskii.cfg" "Apple Macintosh" ".txt"
+    addSystem 1 "$md_id" "macintosh" "$md_inst/bin/BasiliskII --rom $romdir/macintosh/mac.rom --disk $romdir/macintosh/disk.img -- extfs $romdir/macintosh --config $configdir/macintosh/basiliskii.cfg" "Apple Macintosh" ".txt"
```

Comment:
You have a space between `--` and `extfs`

You can test changes by manually running the configure

```
sudo ./retropie_packages.sh basilisk configure
```

if you make the above correction, it will need to be squashed before merge. https://github.com/ginatrapani/todo.txt-android/wiki/Squash-All-Commits-Related-to-a-Single-Issue-into-a-Single-Commit